### PR TITLE
style(valuecard): trend text should be grey instead of indicator color

### LINKED ISSUE #2377: [ValueCard] Trend text should be grey instead of indicator color
### What package is this for?

- [ ] React

### Summary

Text string should be grey, instead of the colors of the ^ indicators
Screen Shot 2021-06-07 at 11 28 02 AM

From design https://ibm.invisionapp.com/share/QMO0YU843D2#/screens/319977464
Screen Shot 2021-06-08 at 12 35 50 PM

Currently, text takes the trend color
Screen Shot 2021-06-08 at 12 33 53 PM

### PR DESCRIPTION
**Pull request by kalyanixraut**
Closes #2377

**Summary**

- The secondary-value-color in ValueCard should be grey(#6F6F6F), instead of the color of the ^ indicator.
- The indicator color when not specified will be display in #6F6F6F color.
- Font size of the secondary value is updated based on design https://ibm.invisionapp.com/d/main/#/console/15373465/319977464 

**Change List (commits, features, bugs, etc)**

- Changes done to Attribute component for secondary-value-color and font size changes in ValueCard.

**Acceptance Test (how to verify the PR)**

- Go to /?path=/story/1-watson-iot-valuecard--with-trends and verify that the secondary value is displayed in grey(#6F6F6F) whereas the trend indicator icon is displayed in the color as specified in the knob- content.attributes[0].secondaryValue.color

**Regression Test (how to make sure this PR doesn't break old functionality)**

- Changes are related to style so go to /?path=/story/1-watson-iot-valuecard--with-trends and verify all functionalities which should work as before.

StoryBook: https://deploy-preview-2878--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-valuecard--with-trends

**Things to look for during review**

- [ ] Make sure all references to `iot` or `bx` class prefix is using the prefix variable
- [ ] (React) All major areas have a `data-testid` attribute. New test ids should have test written to ensure they are not changed or removed.
- [ ] UI should be checked in RTL mode to ensure the proper handling of layout and text.
- [ ] All strings should be translatable.
- [ ] The code should pass a11y scans (The storybook a11y knob should show no violations). New components should have a11y test written.
- [ ] Unit test should be written and should have a coverage of 90% or higher in all areas.
- [ ] All components should be passing visual regression test. For new styles or components either a visual regression test should be written for all permutations or the base image updated.
- [ ] Changes or new components should either write new or update existing documentation.
- [ ] PR should link and close out an existing issue

### CODE REVIEW

**Review by kevinsperrine — APPROVED** [review]
Thanks!

File: packages/react/src/components/ValueCard/Attribute.jsx [hunk 1]
```diff
@@ -162,20 +162,22 @@ const Attribute = ({
             data-testid={`${testId}-secondary-value`}
             className={`${BEM_BASE}-secondary-value`}
             style={{
-              '--secondary-value-color': secondaryValue.color || '#777',
+              '--secondary-value-color': '#6F6F6F',
```
Comment (davidicus) [inline packages/react/src/components/ValueCard/Attribute.jsx:165]: We should bring in the carbon color tokens `gray60`. You can import from @carbon/colors package
`import { gray60 } from '@carbon/colors'`

**Review by davidicus — COMMENTED** [review]
Looks good. Just the one comment about using the tokens.

File: packages/react/src/components/ValueCard/Attribute.jsx (hunk 1 above)
Comment (kalyanixraut) [reply packages/react/src/components/ValueCard/Attribute.jsx]: @davidicus ,Thanks for reviewing it.
Code has been updated.

https://deploy-preview-2878--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-valuecard--with-trends

**Review by davidicus — APPROVED** [review]

**Pull request merged**